src/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,121 @@
+@import "../../variables.scss";
+
+.navbar {
+  max-width: 480px;
+  position: relative;
+  .space {
+    position: absolute;
+    bottom: 0px;
+    width: 480px;
+    .spot {
+      transform: translateX(calc(100% * var(--navbar-item-active)));
+      transition: transform 0.5s cubic-bezier(0.5, 1.42, 0.58, 1);
+      width: calc(100% / var(--navbar-items));
+      .dot {
+        background-color: #1bbcf6;
+        border-radius: 50%;
+        height: 12px;
+        left: calc(50% - 6px);
+        position: absolute;
+        bottom: 14px;
+        width: 12px;
+      }
+    }
+  }
+}
+
+.navbar-list {
+  background-color: $light-background;
+  border-radius: 10px;
+  color: $light-text;
+  display: flex;
+  height: 80px;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+
+@media (prefers-color-scheme: dark) {
+  .navbar-list {
+    background-color: $dark-background;
+    color: $dark-text;
+  }
+}
+
+// --- NavbarItem styles ---
+
+.navbar-item {
+  align-items: center;
+  align-self: center;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  width: calc(100% / var(--navbar-items));
+  &-btn {
+    cursor: pointer;
+    background-color: transparent;
+    border: none;
+  }
+  &-icon {
+    transform-origin: bottom left;
+    &.active {
+      animation: jump 0.5s;
+    }
+  }
+  &-icon-container {
+    display: inline-flex;
+    align-items: center;
+    position: relative;
+  }
+  &-icon-overlay {
+    background-color: $light-background;
+    top: 0;
+    height: 100%;
+    mix-blend-mode: color;
+    position: absolute;
+    right: 0;
+    transform-origin: bottom left;
+    width: 100%;
+    &.active {
+      height: 0%;
+      transition: height 0.5s cubic-bezier(0, -0.84, 0.91, 1.03);
+      animation: jump 0.5s;
+    }
+  }
+}
+
+@keyframes fadeOut {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes jump {
+  25% {
+    transform: rotate(12deg);
+  }
+}
+
+.navbar-item-title {
+  display: block;
+  margin-top: 10px;
+  &.active {
```

Comment:
tip: When using "state" classes that can be togglable, I like to call them "is*" to easily know that it's a conditional state. eg `isActive` this is inspired by the [SUIT CSS metholodogy](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#is-stateOfComponent)